gufe/tests/test_custom_json.py

```diff
@@ -93,35 +98,43 @@ def _test_round_trip(self, encoder, decoder):
             assert json_str == json_str_2
 
     def test_round_trip(self):
-        encoder, decoder = custom_json_factory([self.codec])
+        encoder, decoder = custom_json_factory(self.required_codecs)
         self._test_round_trip(encoder, decoder)
 
+    def test_legacy_bytes_uncompressed(self):
+        # NOTE: this can be removed in `gufe` 2.0, but is also somewhat harmless
```

Comment:
Please raise an issue with the 2.0 milestone for this.

Reply:
https://github.com/OpenFreeEnergy/gufe/issues/449 opened.